helix-core/src/main/java/org/apache/helix/monitoring/mbeans/ResourceMonitor.java

```diff
@@ -70,6 +71,7 @@ public enum RebalanceStatus {
   // Counters
   private SimpleDynamicMetric<Long> _successfulTopStateHandoffDurationCounter;
   private SimpleDynamicMetric<Long> _successTopStateHandoffCounter;
+  @Deprecated
```

Comment:
do you want to add comment as to we are deprecating this and replacing it with another one, so that others have context on why we marked them for deprecation